### PR TITLE
 update oc cluster up to use openshift-controller-manager-operator

### PR DESCRIPTION
**Pull request by deads2k**
uses https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/1 builds on https://github.com/openshift/origin/pull/21027/

/assign @mfojtik

### CODE REVIEW

**Comment by deads2k** [comment]
It doesn't win any speed records and it highlights the issues we have readiness on controllers, but it eventually settles.

**Comment by deads2k** [comment]
/hold

this can't merge until we have images.

File: install/cluster-openshift-controller-manager-operator/install.yaml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+- apiVersion: openshiftcontrollermanager.operator.openshift.io/v1alpha1
+  kind: OpenShiftControllerManagerOperatorConfig
+  metadata:
+    name: instance
+  spec:
+    managementState: Managed
+    imagePullSpec: openshift/origin-hypershift:latest
+    version: 3.11.0
+    logging:
+      level: 4
```
Comment (tnozicka) [inline install/cluster-openshift-controller-manager-operator/install.yaml]: nit: the usual missing `\n` at the end of the file :)
@deads2k `Settings → Editor → General → Ensure line feed at file end on Save`

Comment (deads2k) [reply install/cluster-openshift-controller-manager-operator/install.yaml]: > nit: the usual missing `\n` at the end of the file :)
> @deads2k `Settings → Editor → General → Ensure line feed at file end on Save`

Set just for you in at least one workspace.  If I lose a long job I'll repush.

**Pull request merged**